### PR TITLE
Create freshworks_crm.json

### PR DESCRIPTION
**Pull request by michaes-jivrus**
Adding source for Freshworks CRM

### CODE REVIEW

**Comment by michaes-jivrus** [comment]
Could you please review and merge this new source for Freshworks CRM?

**Comment by michaes-jivrus** [comment]
@pfrisella @castroalves @mattyb @lmanolov @paramaw 
appreciate it if one of you could review and merge this

File: sources/freshworks_crm.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "FRESHWORKS-CRM",
```
Comment (pfrisella) [inline sources/freshworks_crm.json:2]:
```suggestion
  "id": "FRESHWORKS_CRM",
```

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Comment by michaes-jivrus** [comment]
@pfrisella I have made the changes and pushed the code. Please check

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**